azure-cli.pyproj

```diff
@@ -925,17 +947,7 @@
     <Content Include="command_modules\azure-cli-role\tests\recordings\test_sp_create_scenario.yaml" />
   </ItemGroup>
   <ItemGroup>
-    <Interpreter Include="..\env\">
-      <Id>{54f4b6dc-0859-46dc-99bb-b275c9d0aca3}</Id>
-      <BaseInterpreter>{2af0f10d-7135-4994-9156-5d01c9c11b7e}</BaseInterpreter>
-      <Version>3.5</Version>
-      <Description>env (Python 3.5)</Description>
-      <InterpreterPath>Scripts\python.exe</InterpreterPath>
-      <WindowsInterpreterPath>Scripts\pythonw.exe</WindowsInterpreterPath>
-      <LibraryPath>Lib\</LibraryPath>
-      <PathEnvironmentVariable>PYTHONPATH</PathEnvironmentVariable>
-      <Architecture>X86</Architecture>
-    </Interpreter>
+    <InterpreterReference Include="{6ad52464-4277-4828-ac39-b395ab8744a0}\3.5" />
```

Comment:
Was this opened with VS2017?

Reply:
You need to revert these changes. You can use VS2017, but you need to open the azure-cli2017.sln file (which would modify the azure-cli2017.pyproj file.